src/libraries/System.Formats.Cbor/tests/Writer/CborWriterTests.Simple.netcoreapp.cs

```diff
@@ -18,7 +18,7 @@ public partial class CborWriterTests
         [InlineData(0.00006103515625, "f90400")]
         [InlineData(-4.0, "f9c400")]
         [InlineData(float.PositiveInfinity, "f97c00")]
-        [InlineData(float.NaN, "f9fe00")]
```

Comment:
Could you also add a few reader tests validating that the previously used encoding is still readable as NaN?